src/javascript/binary/websocket_pages/cashier/payment_agent_list.js

```diff
@@ -1,120 +1,115 @@
-const showLoadingImage = require('../../base/utility').showLoadingImage;
-const Cookies          = require('../../../lib/js-cookie');
-const Content          = require('../../common_functions/content').Content;
 const url_for_static   = require('../../base/url').url_for_static;
+const Content          = require('../../common_functions/content').Content;
+const Cookies          = require('../../../lib/js-cookie');
 
-const PaymentAgentListWS = (function() {
+const PaymentAgentList = (() => {
     'use strict';
 
-    let hiddenClass,
-        ddlCountriesID,
-        $paListContainer,
-        residence,
-        agentTemplate;
+    let $pa_list_container,
+        $agent_template,
+        ddl_countries_id;
 
+    const hidden_class = 'hidden';
 
-    const init = function() {
-        $(function() {
+    const onLoad = () => {
+        Content.populate();
+        $(() => {
             $('#accordion').accordion({
                 heightStyle: 'content',
                 collapsible: true,
                 active     : false,
             });
         });
 
-        hiddenClass = 'hidden';
-        ddlCountriesID = '#target_country';
-        $paListContainer = $('#pa_list');
-        agentTemplate = $paListContainer.find('#accordion').html();
+        ddl_countries_id   = '#target_country';
```

Comment:
You can declare this value above, similar to hidden_class, since it's a string and not a DOM or jQuery object